### DISCUSSION TITLE
stories.json missing from Storybook 8 (composition)

### DISCUSSION THREAD

**Discussion by marksy** [storybookjs/storybook#29553] [Help]
### Summary

I'm using composition to show two instances of Storybook.

I'm hosting them on my private GitHub Pages, however it seems to be requesting stories.json and when I check my local build folder it's not in there.

I read that the previous version of Storybook had the ability to buildStoriesJson: 

```
features: {
        buildStoriesJson: true,
    },
```

But I think this isn't supported in v8.

**Comment by marksy** [comment]
Oh! unless it's been renamed to index.json 🤔 

Storybook seems to still be requesting stories.json though

**Reply by levrik** [reply]
I'm facing the same issue. Were you able to solve it? File is served as `index.json` but Storybook requests `stories.json`...
